.azure-pipelines/Linux-CI.yml

```diff
@@ -11,23 +11,33 @@ jobs:
         python.version: '2.7'
         onnx_ml: 0
         onnx_debug: 0
+        onnx_lite: "OFF"
+      py27-lite:
+        python.version: '2.7'
+        onnx_ml: 0
+        onnx_debug: 0
+        onnx_lite: "ON"
       py36:
         python.version: '3.6'
         onnx_ml: 0
         onnx_debug: 0
+        onnx_lite: "OFF"
       py37:
         python.version: '3.7'
         onnx_ml: 0
         onnx_debug: 0
+        onnx_lite: "OFF"
       py37-ml:
         python.version: '3.7'
         onnx_ml: 1
         onnx_debug: 0
+        onnx_lite: "OFF"
```

Comment:
this still did not take effect.... I think you need single quotation 'OFF'

Reply:
Thanks for letting me know! Will try that.